Model/Config/Repository.php

```diff
@@ -156,6 +156,14 @@ public function isOrderIntentEnabled(?int $storeId = null): bool
         return $this->isSetFlag(self::XML_PATH_ENABLE_ORDER_INTENT, $storeId);
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function isMultipleInvoiceEmailsEnabled(?int $storeId = null): bool
```

Comment:
nitpick: can we please drop the "Multiple" part? Gives that part too much emphasis IMO :D this is just about whether to enable invoice email or not.